controls/radgridview/events/column-resize-event.md

```diff
@@ -150,13 +150,13 @@ The following example shows how to allow the user to widen the column, bot not s
 	End Sub
 	{{endregion}}
 
->When the user __double clicks__ the header cell gripper and the column is resized according to its content, only the __ColumnWidthChanged__ event is fired.
+>When the user __double-clicks__ the header cell gripper and the column is resized according to its content, only the  __ColumnWidthChanged__ event is triggered.
```

Comment:
**Regarding Comment 9:** What I meant is "When the user __double-clicks__ the header cell gripper and the column is resized according to its content, only the  __ColumnWidthChanged__ event is triggered (the ColumnWidthChanging event is not triggered)."